pkg/nodeconfig/nodeconfig.go

```diff
@@ -152,6 +153,9 @@ func (nc *nodeConfig) Configure() error {
 	if err := nc.createBootstrapFiles(); err != nil {
 		return err
 	}
+	if err := nc.createRegistryConfigFiles(); err != nil {
```

Comment:
we are creating the Containerd directory inside nc.Windows.Boostrap(), why don't we call this function after it is created? If needed, we can also split the bootstrap function into two. Let me know if there is another reason for doing this here.

Reply:
My thoughts here were that createBootstrapFiles() sets up files needed for kubelet and createRegistryConfigFiles() basically does the same for containerd (and containerd is a pre-req service to kubelet) so the funcs should be called together

> we are creating the Containerd directory inside nc.Windows.Boostrap()

This is a good point though, I will explore your suggestion

Reply:
I thought about this more and decided to keep it where it is (though I can still be swayed)
Additional reasoning was 
- this way the registry config is created before containerd service actually starts so it doesn't have to do any additional config loading during runtime
- createRegistryConfigFiles() creates the required directories if needed so no issue